src/migrations/20201020143658-create-trip.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
       returnDate: {
         type: Sequelize.DATE
       },
-      AccommodationId: {
+      accommodationId: {
```

Comment:
@CYIMANA-Faisal create a new migration file that changes the name of this column